data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3611,6 +3611,11 @@ spec:
                         - message: type is immutable once set
                           rule: oldSelf == '' || self == oldSelf
                     type: object
+                  prismAPICallTimeout:
```

Comment:
Generally we don't add configurable timeouts this way. We have used some wait-until-complete type timeouts. Does this need to be configurable or can we just increase the timeout?

Reply:
@barbacbd This PR is manually cherrypick-ed from the PRs https://github.com/openshift/installer/pull/9406 (for release-4.17), https://github.com/openshift/installer/pull/9399 (for release-4.18) and https://github.com/openshift/installer/pull/9377 (for master branch).